deploy/helm/sumologic/upgrade-2.0.0.sh

```diff
@@ -386,6 +386,18 @@ function add_new_scrape_labels_to_prometheus_service_monitors(){
     'prometheus-operator.prometheus.additionalServiceMonitors.[*].selector.matchLabels."sumologic.com/scrape"' true
 }
 
+function kube_prometheus_stack_set_remote_write_timeout_to_5s() {
+  if [[ -z "$(yq r "${TEMP_FILE}" -- 'kube-prometheus-stack.prometheus.prometheusSpec.remoteWrite')" ]]; then
+    return
+  fi
+
+  if [[ -n "$(yq r "${TEMP_FILE}" -- 'kube-prometheus-stack.prometheus.prometheusSpec.remoteWrite.[*].remoteTimeout')" ]]; then
+    echo
+    info "kube-prometheus-stack.prometheus.prometheusSpec.remoteWrite.[*].remoteTimeout is set"
+    info "Please note that we've set it by default to 5s in 2.0.0"
+  fi
```

Comment:
We should inform if `remoteTImeout` is not set as well

Reply:
cc @perk-sumo

Reply:
or set it

Reply:
It is set by default in values.yaml, no need to add this into migrated conf file.